amundsen_application/static/js/components/SearchPage/constants.ts

```diff
@@ -6,7 +6,9 @@ export const DOCUMENT_TITLE_SUFFIX = ' - Amundsen Search';
 
 export const PAGE_INDEX_ERROR_MESSAGE = 'Page index out of bounds for available matches';
 
-export const SEARCH_INFO_TEXT = `Ordered by the relevance of matches within a resource's metadata, as well as overall usage.`;
+export const SEARCH_INFO_TEXT_SUFFIX = `Ordered by the relevance of matches within a resource's metadata`;
+export const SEARCH_INFO_TEXT_TABLE_PREFIX = ', as well as overall usage';
+
 export const SEARCH_SOURCE_NAME = 'search_results';
 export const SEARCH_ERROR_MESSAGE_INFIX = ' - did not match any ';
 export const SEARCH_ERROR_MESSAGE_PREFIX = 'Your search - ';
```

Comment:
Similarly we could use `[prefix] [search term] [suffix]` here instead of infix.

Reply:
will cleanup